### DISCUSSION TITLE
Ability to create custom policies regarding version check

### DISCUSSION THREAD

**Discussion by pedgarcia** [DependencyTrack/dependency-track#1451] [Q&A] [answered]
In my organization, one of the things we want to track is "how old" a certain node package is. We want to have thresholds that alert us based on criteria like:

- Flag it after X major versions 
- Flag it after Y minor versions

Our concern is to make sure we don't get too behind in certain packages, independently of public vulnerabilities found in these packages.

Is this possible?

Thanks

Paulo

**Comment by stevespringett** [comment, marked as answer]
We have plans to support policy evaluation based on component age. See #772.

There are a few discussions on how to support policies based on components being out of date. See #208 and #257.
